src/main/java/cz/brmlab/yodaqa/analysis/passage/CandidateGenerator.java

```diff
@@ -79,15 +79,15 @@ protected void addCandidateAnswer(JCas passagesView, Passage p, Annotation np, A
 			// this should be a singleton
 			break;
 		}
-
 		CandidateAnswer ca = new CandidateAnswer(passagesView);
```

Comment:
Why are we removing this empty line? Seems to me it separates two logical blocks of code.